Formula/ssh-agent-filter.rb

```diff
@@ -0,0 +1,47 @@
+class SshAgentFilter < Formula
+  desc "filtering proxy for ssh-agent"
+  homepage "https://github.com/tiwe-de/ssh-agent-filter"
+  url "https://github.com/tiwe-de/ssh-agent-filter/archive/0.4.2.tar.gz"
+  sha256 "5586e40da34f4afd873d7ff30f115c4a1b9455c1b81f3c4679c995db7615b080"
+
+  depends_on "help2man" => :build
+  depends_on "pandoc" => :build 
+  depends_on "boost"
+  depends_on "nettle"
+
+  patch do
+    # https://github.com/tiwe-de/ssh-agent-filter/pull/6
+    url "https://patch-diff.githubusercontent.com/raw/tiwe-de/ssh-agent-filter/pull/6.diff"
```

Comment:
Use `https://github.com/tiwe-de/ssh-agent-filter/pull/6.diff` as the URL to remove the need for the comment.